source/core/index.rst

```diff
@@ -37,12 +41,19 @@ it's up to the player to find the one that suits them the most.
    placed kicks and punches to block chi paths in bender's bodies, doing
    anything from preventing them from bending to temporarily paralyzing
    them.
+   
+:ref:`Commands and Permissions <commands>`
```

Comment:
Referencing an explicit link puts the title, don't need to write it out yourself

Reply:
Yes it does, however I wrote out all of my references incase I felt that it should be changed